Modix.Services/Core/AuthorizationAutoConfigBehavior.cs

```diff
@@ -3,6 +3,7 @@
 
 using Discord;
 using Discord.WebSocket;
+using Modix.Services.Moderation;
```

Comment:
Why was this added?

Reply:
This was added due to the comments complaining that it required the reference

Reply:
Oh, I see it. Yeah, that's actually copypasta. The comment should be referencing the class itself, `AuthorizationAutoConfigBehavior`.

Reply:
No problemo, I'll make the change for this